src/app/models/treeoperations.cpp

```diff
@@ -157,3 +157,19 @@ void TreeOperations::notifyDbWasUnloaded(int dbIndex)
 {
     emit closeDbKeys(m_connection, dbIndex);
 }
+
+void TreeOperations::deleteDbKey(int dbIndex, ConnectionsTree::KeyItem& key)
```

Comment:
You do not need `dbIndex` arg here because you can get it from `key` arg - https://github.com/uglide/RedisDesktopManager/blob/0.8.0/src/modules/connections-tree/items/keyitem.h#L31